client/blocks/nps-survey/docs/example.jsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import NpsSurvey from '../';
+import { isEnabled } from 'config';
+
+class NpsSurveyExample extends PureComponent {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			wasDismissed: false,
```

Comment:
Is there any reason to not just have the initial state be an empty object here? We're not trying to reference anything that'd fatal below that I can tell.